### ISSUE TITLE
localeResolutionCallback doesn't work correctly

### ISSUE THREAD

**Issue by bbrto21** [flutter-tizen/engine#36]
**Problem**
  - If you use localeResolutioncallback, you get the wrong locale value
  - Sample codes
      ```dart
      // ...
      import 'package:flutter_localizations/flutter_localizations.dart';
      // ...
            localizationsDelegates: [
              GlobalMaterialLocalizations.delegate,
              GlobalWidgetsLocalizations.delegate,
              GlobalCupertinoLocalizations.delegate,
            ],
            localeResolutionCallback:
                (Locale locale, Iterable<Locale> supportedLocales) {
              print("localeResolutionCallback!!");
              print("locale.countryCode : " + locale.countryCode.toString());
              print("locale.languageCode : " + locale.languageCode.toString());
              return locale;
           }
      ```
**How to reproduce**
1. Launch the app 
2. Change device language to another language
3. Resume the app

**as is** 
 - The locale value is always wrong.
![image](https://user-images.githubusercontent.com/1579532/106848164-c13cb780-66f3-11eb-93fc-656e10f25f7e.png)

**expected**
 - The locale value should reflect the language of the device.
![image](https://user-images.githubusercontent.com/1579532/106847918-38be1700-66f3-11eb-9ed6-4f559352c911.png)

**Comment by swift-kim** [comment]
I had the same issue Gallery app being displayed in Afrikaans on my emulators.

**Comment by bbrto21** [comment]
> I had the same issue Gallery app being displayed in Afrikaans on my emulators.

I also started by analyzing the gallery app and I guess that there are some problems.
  - First, it doesn't reflect the device language 
  - Second, it seems that the fallback font is not selecte properly (I haven't found a clear reason yet.)
It seems that CJK cannot be displayed properly due to the above two problems.

**Comment by bbrto21** [comment]
> Second, it seems that the fallback font is not selecte properly (I haven't found a clear reason yet.)

we need to check whether the text is displayed properly when specify the font family including CJK explicitly

**Comment by swift-kim** [comment]
It seems there are some fallback fonts under `/usr/share/fallback_fonts` that can be used to display non-alphabetic characters. Apparently SamsungOneUI or BreezeSans (in `/usr/share/fonts`) isn't enough to display all unicode characters. Still I don't fully understand how font resolution works on Tizen and inside the Flutter SDK. It was more complicated than I thought when I went through the underlying code before.

**Comment by bbrto21** [comment]
if #38 is merged, flutter get a device locale properly

**Comment by bbrto21** [comment]
> It seems there are some fallback fonts under `/usr/share/fallback_fonts` that can be used to display non-alphabetic characters. Apparently SamsungOneUI or BreezeSans (in `/usr/share/fonts`) isn't enough to display all unicode characters. Still I don't fully understand how font resolution works on Tizen and inside the Flutter SDK. It was more complicated than I thought when I went through the underlying code before.

I will open an issue for this

**Comment by bbrto21** [comment]
Resolved by #38
